resources/packs/azure/azure.lr.manifest.yaml

```diff
@@ -1013,6 +1013,8 @@ resources:
       name: {}
       osDisk: {}
       properties: {}
+      publicIpAddress: {}
```

Comment:
I'm curious what this is for.
When I try to use it in the shell, it fails:
```
cnquery> azure.subscription.compute.vms{name publicIpAddress{*}}
failed to compile: cannot find field or resource 'publicIpAddress' in block for type 'azure.subscription.computeService.vm'
```

Reply:
Great catch, that was a leftover from developing